config.py

```diff
@@ -18,36 +17,39 @@
 
 plot = False
 
+
+# ``kind`` : str
+#   The kind of data to be analyzed, can be 'eeg', or 'meg'.
+
+kind = 'eeg'
+
+# ``datatype`` : str
+#   The file ending, can be '.fif', '.vhdr', '.ds', ...
+dataformat = '.vhdr'
```

Comment:
Thinking about it, you shouldn't need this. There should be only file in the BIDS folder that is stored for a particular session, task and run, right? Does BIDS allow me to store both `sub-01_ses-01_task-audvis_run-01.fif` and `sub-01_ses-01_task-audvis_run-01.vhdr`

Reply:
no, but BrainVision,e.g., has `vmrk, vhdr, eeg` :thinking:

Reply:
sure but in the end you read just one file that is uniquely identified without the need for file extension ...